src/rgw/rgw_request.h

```diff
@@ -59,9 +59,10 @@ struct RGWLoadGenRequest : public RGWRequest {
 	string method;
 	string resource;
 	int content_length;
-	std::atomic<int64_t>* fail_flag = nullptr;
+	std::atomic<bool>* fail_flag = { nullptr };
```

Comment:
In other conversation, it was agreed that we might as well convert "flags" to atomic<bool> while we're in here.